### ISSUE TITLE
Submitting replication done as the paper reviewer

### ISSUE THREAD

**Issue by leouieda** [ReScience/ReScience#23]
I have been trying to do implementations of methodological work I'm asked to peer-review for journals. This is not always possible but I have managed to do a couple and successfully reproduce the submitted results. I shared my reproduction as part of my review and the papers were accepted and published.

Is there any ethical issue with submitting my reproduction made during peer-review after the article has been accepted and published?

**Comment by rougier** [comment]
I'm not sure to understand. Correct me if I'm wrong:

You were a reviewer for a paper and, as part of your review, you reproduced the work by writing your own version of the method or result ? If your implementation is your own work (meaning, you did not take code from the original paper), I don't see any ethical issue.

@khinsen what do you think ?

**Comment by khinsen** [comment]
I completely agree. As a reviewer you are supposed to treat the submission you are reviewing as confidential, but once it is accepted and published, you are free to talk about it in any way you like.

There might be a legal issue if you signed some form of copyright transfer for your review. That has never happened to me, but then I never reproduced any computation as part of a review either!

**Comment by leouieda** [comment]
> You were a reviewer for a paper and, as part of your review, you reproduced the work by writing your own version of the method or result?

@rougier that is exactly the case. 

The implementation is my own and I didn't sign any copyright transfer agreements with the journal. So I guess it would be OK to submit the reproduction. I just thought I should ask to make sure there are no issues. 

I look on the ReScience FAQ but there was nothing there. Maybe this is something worth including? Could be an incentive for trying to reproduce the work during review (you can get a paper out of it, so it's not "wasted" effort).

**Comment by rougier** [comment]
Yes, we can add (even if I'm not sure this is such a frequent case actually). Would mind making a PR for the FAQ (else I can do it later but I'm on the leave)?

And of course, you're more than welcome to submit your work.

**Comment by leouieda** [comment]
I can open the PR. Should it be in the ReScience/rescience.github.io repo? I could put this somewhere else if the FAQ is not appropriate.

**Comment by rougier** [comment]
Yes, rescience.github.io. And I think the FAQ is the proper place.
By the way, is it necessary to talk about the copyright transfer agreements as reviewer? I never had something like that. Did you ever experienced that?
